libraries/google_compute_firewalls.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'gcp_backend'
+
+module Inspec::Resources
+  class GoogleComputeFirewalls < GcpResourceBase
+    name 'google_compute_firewalls'
+    desc 'Verifies settings for GCP compute firewalls in bulk'
+
+    example "
+      describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+        it { should exist }
+        ...
+      end
+    "
+
+    def initialize(opts = {})
+      # Call the parent class constructor
+      super(opts)
+      @display_name = opts[:name]
+      @project = opts[:project]
+    end
+
+    # FilterTable setup
+    filter_table_config = FilterTable.create
+    filter_table_config.add_accessor(:where)
+    filter_table_config.add_accessor(:entries)
+    filter_table_config.add(:exist?) { |filter_table| !filter_table.entries.empty? }
+    filter_table_config.add(:count) { |filter_table| filter_table.entries.count }
```

Comment:
With https://github.com/inspec/inspec/pull/3104 we don't need to add `where`, `entries`, `exist?` or `count` properties to filter tables, they are now automatic.

Reply:
Happy to do this but worth noting all profiles using inspec-gcp will now have to update the minimum InSpec version because of https://github.com/inspec/inspec/issues/3066